Currently inductor doesn't look in conda's includes and libs. This results in errors when it tries to compile, if system versions are being used of dependencies (e.g., sleef).

author: dpetry@anaconda.com

Index: pytorch/torch/_inductor/cpp_builder.py
===================================================================
diff --git torch/_inductor/cpp_builder.py torch/_inductor/cpp_builder.py
index d0a646c..7468f27 100644
--- a/torch/_inductor/torch/_inductor/cpp_builder.py
+++ b/torch/_inductor/torch/_inductor/cpp_builder.py
@@ -1136,6 +1136,7 @@ def get_cpp_torch_options(
         + python_include_dirs
         + torch_include_dirs
         + omp_include_dir_paths
+        + [os.getenv('CONDA_PREFIX') + '/include']
     )
     cflags = sys_libs_cflags + omp_cflags
     ldflags = omp_ldflags
